lib/ansible/modules/packaging/os/portage.py

```diff
@@ -462,7 +462,7 @@ def run_emerge(module, packages, *args):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            package=dict(default=None, aliases=['name'], type='list'),
+            package=dict(type='list', default=None, aliases=['name']),
```

Comment:
You didn't add elements='str' here, was it intended ?

Reply:
Yep! I didn't have apportunity to test on Gentoo and just fix style :)

Reply:
Sorry, I did not realize this affected the portage module. I can see about testing if others do not have time. Offhand, I do not see it causing any issues.

Reply:
You are right. It's just my first contribution and I'm little cowardly. I will check if this change is needed for other modules.

Reply:
I add option for portage and homebrew\macports.